composer.json

```diff
@@ -22,7 +22,8 @@
         "symfony/filesystem": "^3.4|^4.0|^5.0",
         "symfony/finder": "^3.4|^4.0|^5.0",
         "symfony/framework-bundle": "^3.4|^4.0|^5.0",
-        "symfony/http-kernel": "^3.4|^4.0|^5.0"
+        "symfony/http-kernel": "^3.4|^4.0|^5.0",
+        "symfony/string": "^5.0"
```

Comment:
There's a practical problem that I don't know if we can overcome. If a project uses Symfony 4.4, and has `extra.symfony.require` set to `4.4.*`, I don't think they will be able to install Maker. They would get an error that no `symfony/string` version was found, because Flex filters it to only match 4.4 (but we should actually try this to be sure).

Reply:
Btw, can we drop 3.4 support for a min version with 4.4 ?

Reply:
Just for the record, looks like I can install `symfony/string:^5.1` with `symfony/maker-bundle:^1.21` + `symfony.extra.require:4.4.*`. But `symfony/string` is a direct dependency 🤔  Will it work with an indirect dependency by `maker-bundle`, too?

![CleanShot 2020-10-23 at 20 34 11](https://user-images.githubusercontent.com/995707/97041047-2e767380-156f-11eb-95eb-57fbb948af71.png)

Reply:
This specific combinations should work because String is not in 4.4, this is not bounded by the extra require constraint.

Reply:
> Btw, can we drop 3.4 support for a min version with 4.4 ?

@LeJeanbono IMO this is not needed, as you can install `3.4` when using `7.2.5` php afterwards 👍